metals/src/main/scala/scala/meta/internal/metals/Compilers.scala

```diff
@@ -841,6 +853,26 @@ class Compilers(
     }
   }
 
+  def loadJavaCompilerForTarget(
+      target: JavaTarget
+  ): Option[PresentationCompiler] = {
+    val pc = JavaPresentationCompiler()
+
+    val name = target.javac.getTarget.getUri
+    val classpath =
+      target.javac.classpath.toAbsoluteClasspath.map(_.toNIO).toSeq
+    val options = target.javac.getOptions.asScala.toSeq
+    val filteredOptions = plugins.filterSupportedOptions(options)
```

Comment:
@VankaTaganai 
It seems that we should remove this line as `filterSupportedOption` is designed only for scala.

This should fix an issue with RenameLspSuite witch @tgodzik mentioned here - https://github.com/scalameta/metals/pull/5024#issuecomment-1551695564